### PR TITLE
[ Xedra Evolved ] Cryptid video snippets

### PR DESCRIPTION
**Pull request by Maleclypse**
#### Summary
Mods "Xedra Evolved Cryptid Video snippets"

#### Purpose of change
Increase lore to be found in XE.

#### Describe the solution
Adds snippets.  Will add more as I write them.  This can be merged now though.

#### Describe alternatives you've considered
Waiting til I have more

#### Testing
yup

### CODE REVIEW

File: data/mods/Xedra_Evolved/snippets/e_logs.json [hunk 1]
```diff
@@ -6,41 +6,67 @@
...
...g from the 90's begins playing.  The young adults enter a classroom and are clearly flirting.  \nAwoo oooh, got you where I want you\nplays over top of the video.  The man laughs at something the woman says and throws his head back laughing while she is clearly annoyed.  His face elongates as his body grows shredding the clothes he is wearing as limbs change and sprout thick fur.  She has backed away from him and put the students and teacher's desks between them.  Fully transformed into a hideous wolfman he begins his approach towards his once date now prey.  Smashing chairs and students desks out of his way.  Suddenly the wolfcreature pauses and ears prick up.  It turns and looks at the wall of windows in the classroom.  Glass erupts into the room as the darkness outside the window shows the muzzle flare of at least a dozen different guns firing.  The wolfcreature is hit more times than you can count in a span of seconds and falls to the ground.  \nThe song reaches a crescendo 'Got you where I want you, got you, yeahhhhh!'\nAs his date pulls a large caliber pistol out of the teachers desk, walks over to the fallen beast and calmly puts three bullets in its skull.\nThe video ends."
```
Comment (Maleclypse) [inline data/mods/Xedra_Evolved/snippets/e_logs.json:63]:
```suggestion
        "text": "hunternetvulfgar.mp4\nThe video cuts in as a montage of school security cameras in hallways as a young man and woman break into the school.  As they walk past each camera that image drops and the others rearange larger while a popular song from the 90's begins playing.  The young adults enter a classroom and are clearly flirting.  \nAwoo oooh, got you where I want you\nplays over top of the video.  The man laughs at something the woman says and throws his head back laughing while she is clearly annoyed.  His face elongates as his body grows shredding the clothes he is wearing as limbs change and sprout thick fur.  She has backed away from him and put the students and teacher's desks between them.  Fully transformed into a hideous wolfman he begins his approach towards his once date now prey.  Smashing chairs and students desks out of his way.  Suddenly the wolfcreature pauses and ears prick up.  It turns and looks at the wall of windows in the classroom.  Glass erupts into the room as the darkness outside the window shows the muzzle flare of at least a dozen different guns firing.  The wolfcreature is hit more times than you can count in a span of seconds and falls to the ground.  \nThe song reaches a crescendo 'Got you where I want you, got you, yeahhhhh!'\nAs his date pulls a large caliber pistol out of the teachers desk, walks over to the fallen beast and calmly puts three bullets in its skull.\nThe video ends."
```

File: data/mods/Xedra_Evolved/snippets/e_logs.json [hunk 2]
```diff
@@ -6,41 +6,67 @@
...
       }
     ]
+  },
+  {
+    "type": "snippet",
+    "category": "local_files_simple",
+    "//": "This is extending the vanilla snippets belonging to mostly work of random people: chat logs, SMSes, popular photos and videos that people share, or something they can film by themselves. ",
```
Comment (Maleclypse) [inline data/mods/Xedra_Evolved/snippets/e_logs.json:58]:
```suggestion
    "//": "This is extending the vanilla snippets belonging to mostly work of random people: chat logs, SMSes, popular photos and videos that people share, or something they can film by themselves.",
```

File: data/mods/Xedra_Evolved/snippets/e_logs.json [hunk 3]
```diff
@@ -6,41 +6,67 @@
...
+      },
+      {
+        "id": "xe_local_files_simple_2",
+        "name": "cryptid sighting vampire",
+        "text": "hunternetlinda.mp4\n\nElectronic music plays on a black screen.  Text appears on the screen\n'Stand still, pause clocks, we can make the world stop'\nA video montage follows a woman across multiple days and outfits.  On errands, at work, traveling in different cities.  About a minute into the montage of video there are short cuts to her violently attacking people in secluded settings.  Those gradually become the dominant images until it's only video of her feasting on the blood of her fallen victims.  The final image is her looking up; face covered, blood dripping down onto her ruined outfit; eyes staring straight into the camera as the image freezes. On the now black screen a name and address appear on it."
```
Comment (Maleclypse) [inline data/mods/Xedra_Evolved/snippets/e_logs.json:68]:
```suggestion
        "text": "hunternetlinda.mp4\nElectronic music plays on a black screen.  Text appears on the screen\n'Stand still, pause clocks, we can make the world stop'\nA video montage follows a woman across multiple days and outfits.  On errands, at work, traveling in different cities.  About a minute into the montage of video there are short cuts to her violently attacking people in secluded settings.  Those gradually become the dominant images until it's only video of her feasting on the blood of her fallen victims.  The final image is her looking up; face covered, blood dripping down onto her ruined outfit; eyes staring straight into the camera as the image freezes. On the now black screen a name and address appear on it."
```

File: data/mods/Xedra_Evolved/snippets/e_logs.json [hunk 4]
```diff
@@ -6,41 +6,67 @@
...
...g from the 90's begins playing.  The young adults enter a classroom and are clearly flirting.  \nAwoo oooh, got you where I want you\nplays over top of the video.  The man laughs at something the woman says and throws his head back laughing while she is clearly annoyed.  His face elongates as his body grows shredding the clothes he is wearing as limbs change and sprout thick fur.  She has backed away from him and put the students and teacher's desks between them.  Fully transformed into a hideous wolfman he begins his approach towards his once date now prey.  Smashing chairs and students desks out of his way.  Suddenly the wolfcreature pauses and ears prick up.  It turns and looks at the wall of windows in the classroom.  Glass erupts into the room as the darkness outside the window shows the muzzle flare of at least a dozen different guns firing.  The wolfcreature is hit more times than you can count in a span of seconds and falls to the ground.  \nThe song reaches a crescendo 'Got you where I want you, got you, yeahhhhh!'\nAs his date pulls a large caliber pistol out of the teachers desk, walks over to the fallen beast and calmly puts three bullets in its skull.\nThe video ends."
```
Comment (Maleclypse) [inline data/mods/Xedra_Evolved/snippets/e_logs.json:63]:
```suggestion
        "text": "hunternetvulfgar.mp4\nThe video cuts in as a montage of school security cameras in hallways as a young man and woman break into the school.  As they walk past each camera that image drops and the others rearange larger while a popular song from the 90's begins playing.  The young adults enter a classroom and are clearly flirting.\nAwoo oooh, got you where I want you\nplays over top of the video.  The man laughs at something the woman says and throws his head back laughing while she is clearly annoyed.  His face elongates as his body grows shredding the clothes he is wearing as limbs change and sprout thick fur.  She has backed away from him and put the students and teacher's desks between them.  Fully transformed into a hideous wolfman he begins his approach towards his once date now prey.  Smashing chairs and students desks out of his way.  Suddenly the wolfcreature pauses and ears prick up.  It turns and looks at the wall of windows in the classroom.  Glass erupts into the room as the darkness outside the window shows the muzzle flare of at least a dozen different guns firing.  The wolfcreature is hit more times than you can count in a span of seconds and falls to the ground.\nThe song reaches a crescendo 'Got you where I want you, got you, yeahhhhh!'\nAs his date pulls a large caliber pistol out of the teachers desk, walks over to the fallen beast and calmly puts three bullets in its skull.  The video ends."
```

File: data/mods/Xedra_Evolved/snippets/e_logs.json [hunk 5]
```diff
@@ -6,41 +6,67 @@
...
+      },
+      {
+        "id": "xe_local_files_simple_2",
+        "name": "cryptid sighting vampire",
+        "text": "hunternetlinda.mp4\nElectronic music plays on a black screen.  Text appears on the screen\n'Stand still, pause clocks, we can make the world stop'\nA video montage follows a woman across multiple days and outfits.  On errands, at work, traveling in different cities.  About a minute into the montage of video there are short cuts to her violently attacking people in secluded settings.  Those gradually become the dominant images until it's only video of her feasting on the blood of her fallen victims.  The final image is her looking up; face covered, blood dripping down onto her ruined outfit; eyes staring straight into the camera as the image freezes. On the now black screen a name and address appear on it."
```
Comment (Maleclypse) [inline data/mods/Xedra_Evolved/snippets/e_logs.json:68]:
```suggestion
        "text": "hunternetlinda.mp4\nElectronic music plays on a black screen.  Text appears on the screen\n'Stand still, pause clocks, we can make the world stop'\nA video montage follows a woman across multiple days and outfits.  On errands, at work, traveling in different cities.  About a minute into the montage of video there are short cuts to her violently attacking people in secluded settings.  Those gradually become the dominant images until it's only video of her feasting on the blood of her fallen victims.  The final image is her looking up; face covered, blood dripping down onto her ruined outfit; eyes staring straight into the camera as the image freezes.  On the now black screen a name and address appear on it."
```

File: data/mods/Xedra_Evolved/snippets/e_logs.json [hunk 6]
```diff
@@ -6,41 +6,67 @@
...
... song from the 90's begins playing.  The young adults enter a classroom and are clearly flirting.\nAwoo oooh, got you where I want you\nplays over top of the video.  The man laughs at something the woman says and throws his head back laughing while she is clearly annoyed.  His face elongates as his body grows shredding the clothes he is wearing as limbs change and sprout thick fur.  She has backed away from him and put the students and teacher's desks between them.  Fully transformed into a hideous wolfman he begins his approach towards his once date now prey.  Smashing chairs and students desks out of his way.  Suddenly the wolfcreature pauses and ears prick up.  It turns and looks at the wall of windows in the classroom.  Glass erupts into the room as the darkness outside the window shows the muzzle flare of at least a dozen different guns firing.  The wolfcreature is hit more times than you can count in a span of seconds and falls to the ground.\nThe song reaches a crescendo 'Got you where I want you, got you, yeahhhhh!'\nAs his date pulls a large caliber pistol out of the teachers desk, walks over to the fallen beast and calmly puts three bullets in its skull.  The video ends."
```
Comment (harakka) [inline data/mods/Xedra_Evolved/snippets/e_logs.json:63]:
```suggestion
        "text": "hunternetvulfgar.mp4\nThe video cuts in as a montage of school security cameras in hallways, as a young man and woman break into the school.  As they walk past each camera, that image drops and the others rearrange larger, while a popular song from the 90's begins playing.  The young adults enter a classroom and are clearly flirting.\nAwoo oooh, got you where I want you\nplays over top of the video.  The man laughs at something the woman says, and throws his head back laughing while she is clearly annoyed.  His face elongates as his body grows, shredding the clothes he is wearing as limbs change and sprout thick fur.  She has backed away from him and put the students' and teacher's desks between them.  Fully transformed into a hideous wolfman, he begins his approach towards his once date now prey, smashing chairs and students' desks out of his way.  Suddenly the wolfcreature pauses and its ears prick up.  It turns and looks at the wall of windows in the classroom.  Glass erupts into the room as the darkness outside the window shows the muzzle flare of at least a dozen different guns firing.  The wolfcreature is hit more times than you can count in a span of seconds and falls to the ground.\nThe song reaches a crescendo 'Got you where I want you, got you, yeahhhhh!' as his date pulls a large caliber pistol out of the teachers desk, walks over to the fallen beast and calmly puts three bullets in its skull.  The video ends."
```
Added some commas to make the sentece structure read clearer and few other minor readability improvements

File: data/mods/Xedra_Evolved/snippets/e_logs.json [hunk 7]
```diff
@@ -6,41 +6,67 @@
...
+      },
+      {
+        "id": "xe_local_files_simple_2",
+        "name": "cryptid sighting vampire",
+        "text": "hunternetlinda.mp4\nElectronic music plays on a black screen.  Text appears on the screen\n'Stand still, pause clocks, we can make the world stop'\nA video montage follows a woman across multiple days and outfits.  On errands, at work, traveling in different cities.  About a minute into the montage of video there are short cuts to her violently attacking people in secluded settings.  Those gradually become the dominant images until it's only video of her feasting on the blood of her fallen victims.  The final image is her looking up; face covered, blood dripping down onto her ruined outfit; eyes staring straight into the camera as the image freezes.  On the now black screen a name and address appear on it."
```
Comment (harakka) [inline data/mods/Xedra_Evolved/snippets/e_logs.json:68]:
```suggestion
        "text": "hunternetlinda.mp4\nElectronic music plays on a black screen.  Text appears on the screen\n'Stand still, pause clocks, we can make the world stop'\nA video montage follows a woman across multiple days and outfits.  On errands, at work, traveling in different cities.  About a minute into the montage of video there are short cuts to her violently attacking people in secluded settings.  Those gradually become the dominant images until it's only video of her feasting on the blood of her fallen victims.  The final image is her looking up; face covered, blood dripping down onto her ruined outfit; eyes staring straight into the camera as the image freezes.  On the now black screen, a name and address appear."
```

File: data/mods/Xedra_Evolved/snippets/radio.json [hunk 8]
```diff
@@ -0,0 +1,19 @@
...
+    "text": [
+      {
+        "id": "xe_radio_1",
+        "name": "voices on the radio:deftones",
+        "text": "A voice cuts in across the static, 'And for the next hour we'll be playing the last Deftones album Through the Green Door.  I wonder if we collectively went through a green door before the world went crazy.'  The radio blasts Deftones music until you lose the station a few songs in."
+      },
+      {
+        "id": "xe_radio_2",
+        "name": "voices on the radio:darkest of the hillside thickets",
+        "text": "You find a faint signal 'This is [unintelligible] bringing you Darkest of the Hillside Thickets new single Don't Talk with the Voice in the Storm!  I'm not really sure how new this single is as I never heard it before the Cataclysm and it showed up in an envelope stapled to the station door one morning after a reality storm.  I suppose there are creepier ways to bring me new music but I don't know what they are and I'm begging you not to enact them."
```
Comment (harakka) [inline data/mods/Xedra_Evolved/snippets/radio.json:15]:
```suggestion
        "text": "You find a faint signal: 'This is [unintelligible] bringing you Darkest of the Hillside Thickets' new single Don't Talk with the Voice in the Storm!  I'm not really sure how new this single is as I never heard it before the Cataclysm and it showed up in an envelope stapled to the station door one morning after a reality storm.  I suppose there are creepier ways to bring me new music but I don't know what they are, and I'm begging you not to enact them."
```

**Review by harakka — COMMENTED** [review]
I hope these improve readability a bit, loving this stuff

**Pull request merged**